deployment/integration-tcs/istio-gateway-sgx-tcs-config.yaml

```diff
@@ -0,0 +1,248 @@
+      - pem: |
+          -----BEGIN CERTIFICATE-----
+          MIIEbDCCAtSgAwIBAgIIOANo/iO8CsMwDQYJKoZIhvcNAQELBQAwVDEdMBsGA1UE
+          ChMUSW50ZWwoUikgQ29ycG9yYXRpb24xMzAxBgNVBAMTKlNHWCBzZWxmLXNpZ25l
+          ZCByb290IGNlcnRpZmljYXRlIGF1dGhvcml0eTAeFw0yMjEyMTYwNjQ3MjdaFw0y
+          MzEyMTYwNjQ3MjdaMFQxHTAbBgNVBAoTFEludGVsKFIpIENvcnBvcmF0aW9uMTMw
+          MQYDVQQDEypTR1ggc2VsZi1zaWduZWQgcm9vdCBjZXJ0aWZpY2F0ZSBhdXRob3Jp
+          dHkwggGiMA0GCSqGSIb3DQEBAQUAA4IBjwAwggGKAoIBgQDyUFLLNrGYUBmFFYst
+          5seTG9HCevFLtETcA6qgUheHbx65bjTd/ohjHtv/Gy3dqk4HC4mhr/hiurzTDgR4
+          X00IhLd0uNtfn2Z3+XOkBHn5Rurtj1ZZ+f3jSVAGJF4bzhBS2zsUJYDw8XCMpdPq
+          fQo89fQ4OW2zOMgyOaoE4tKrwDT+3jBKygA22l2rnUOXYtKbPI8mOsK7BRuIWN4G
+          9ECVNEXsozLafhd8jXuCP89eTII0qdw9mJTYsp0A5U3R0rJ6Xs0qul75lAHSLY0G
+          WlG9c+n8G1lXBXxzvoT2kdCtMyRoiJvAT5hNzgTEeuxkQxiSRL5MVq+Qum9EUS8u
+          9NYK13PoDPHUMejfHlE+n8bs7dAQ1rIJHvuO8uHv49cp5/ta0Bd8ggNvNAubMw/A
+          9iQYNNuwbQ2EvaEID02n175xefVEUv/lUfyZ7Qu2CEcN+FfXdCj38DbrZ/Z+hnZv
+          oPqFxobf3wW6zFJXMGNlhJfyMmd+OO8+tUPTXtKZu0sbx2ECAwEAAaNCMEAwDgYD
+          VR0PAQH/BAQDAgKkMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFIC7hkPuHPAu
+          c78eqYeOiQ2XdtrYMA0GCSqGSIb3DQEBCwUAA4IBgQA6mcyDUp3ForwHNxWN9gqK
+          93AEuDAGNI6HGzz5lt/Bfy7am23/Af/WttHtrTeZd8AU32wjH2Tn5x4aLwaDw5QR
+          D2EK2lU5ziBaoQicjtI512Y0AEJodFiXvXztixhsGZdLtifbuNOlXZ4EoWV0g6e/
+          XEko2MQQ9YLD+WbYhlsc2WRux95N25rJU5jzSUT1Y32W9/ODmuA7jzrGmYKWzMLU
+          NKNj2bXR4tnLTHduwHD1pCohP5qLkYBLNY720hltZamQl2LPciGCprnnHQrUKKyC
+          fuvTib/RCS5/qUry58cHfSpdQis1A7MlQspiGggFQf7pqsVfhZtoY1SVvOx/Dzyo
+          9Agfb0dzE11n8UjXoDQwAmdeWLbfw6R3uLVtPlwdgq7n4JNE6XVFBxWwLd+Pt/Zo
+          UTvTPM8ezRiVplq1beuQXVRHH1Bzxedq1R02v/DLkH0dMeSyMU1VS3+HRgOeHXWr
+          ccpv5ngljFfyux9TknYH8Eb85N1Io7JA/Vm9eYv9MJE=
+          -----END CERTIFICATE-----
+        certSigners:
+        - tcsclusterissuer.tcs.intel.com/sgx-signer
+  values:
+    global:
+      imagePullPolicy: IfNotPresent
+    sidecarInjectorWebhook:
+      templates:
+        sgx: |
+          metadata:
+            annotations:
+              sgx.intel.com/quote-provider: aesmd
+          spec:
+            containers:
+            - name: sdsserver
+              image: registry.fi.intel.com/huailong/sds-server:tcs 
+              imagePullPolicy: IfNotPresent
+              lifecycle:
+                postStart:
+                  exec:
+                    command: ["/bin/sh", "-c", "./home/istio-proxy/prepare.sh"]
+              resources:
+                limits:
+                  cpu: 200m
+                  memory: 500Mi
+                  sgx.intel.com/enclave: 1
+                  sgx.intel.com/epc: 1Mi
+                requests:
+                  cpu: 200m
+                  memory: 300Mi
+                  sgx.intel.com/enclave: 1
+                  sgx.intel.com/epc: 1Mi
+              env:
+              - name: NEED_QUOTE
+                value: "false"
```

Comment:
`NEED_QUOTE` and `RANDOM_NONCE` should be `true` since we need quote attestation and using v1alpha2 api.

Reply:
okay, done, make sense